### PR TITLE
Category Feature

### PR DESCRIPTION
**Pull request by Bobj2018**
# Description
The goal was to allow users to add/edit/delete categories and to assign todos to specific categories.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts
- [ ] I have checked that no one else is on staging and deployed this feature branch

### CODE REVIEW

File: models/households-model.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ const insert = (newHousehold) => {
 
 const update = (id, updates) => {
   return db('households')
-    .where({ id })
```
Comment (BenHall-7) [inline models/households-model.js:17]: Why is this change here? Doesn't it need an id to specify which household?

Comment (Bobj2018) [reply models/households-model.js:17]: I'm not entirly sure
